asv/plugins/virtualenv.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, conf, python, requirements, tagged_env_vars):
                                          tagged_env_vars)
 
         try:
-            import virtualenv # noqa F401 unused, but required in case the previous step does not work
+            import virtualenv # noqa F401 unused, but is needed even if not used
```

Comment:
I think what you're saying here doesn't add much value. It's clearly needed if unsused, that's why we have the noqa. Would be more useful to explain why is needed.

Reply:
This line has an import, which is unused. We don't want to have unused imports in general, there is no point. In this case we're going to leave it, and we're telling flake8 this is fine, we want to leave an unused import. That's fine, but let readers of this code know why we're leaving this import. And saying "because it's needed" it's like not saying anyting. Please add here a comment on why this import is needed. In this case it's quite obvious, and could not even be needed, but a short sentence to help readers would still be nice.